pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -506,6 +506,28 @@ func Convert_v1_FeatureHyperv_To_api_FeatureHyperv(source *v1.FeatureHyperv, hyp
 	return nil
 }
 
+func getSriovPciAddresses() []string {
```

Comment:
Can you please add a comment to explain how the `SRIOV-VF-PCI-ADDR` env variable looks like?